### PR TITLE
[DC] Improve search coverage by adding Bing, Google feeds

### PR DESCRIPTION
**Pull request by caiismyname**
## Summary
*  Improve search coverage by adding Bing, Google feeds
* Also, cleanup google feed URLs 

## Testing Plan
* Local running -- does not affect VM builds

## Affected Services
* DC

## Reviewers

### CODE REVIEW

File: DataCollection/data_collection/collecting_data/pull_feeds.py [hunk 1]
```diff
@@ -38,7 +38,6 @@ def parse(self):
             title = entry["title"]
             description = entry["description"]
             coverage_date = entry["published"]
-            authors = [a['name'] for a in entry["authors"] if 'name' in a]
```
Comment (brgirardeau) [inline DataCollection/data_collection/collecting_data/pull_feeds.py]: Any reason you got rid of authors?

Comment (caiismyname) [reply DataCollection/data_collection/collecting_data/pull_feeds.py]: I hate writers

Comment (caiismyname) [reply DataCollection/data_collection/collecting_data/pull_feeds.py]: jk we're not keeping track, and it was giving problems on same feeds

Comment (brgirardeau) [reply DataCollection/data_collection/collecting_data/pull_feeds.py]: 👍

**Pull request merged**